ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/PlayerFragment.kt

```diff
@@ -174,8 +177,10 @@ class PlayerFragment :
         playlistFlipper = view.findViewById(R.id.current_playing_playlist_flipper)
         emptyTextView = view.findViewById(R.id.playlist_empty)
         songTitleTextView = view.findViewById(R.id.current_playing_song)
-        albumTextView = view.findViewById(R.id.current_playing_album)
         artistTextView = view.findViewById(R.id.current_playing_artist)
+        albumTextView = view.findViewById(R.id.current_playing_album)
+        genreTextView = view.findViewById(R.id.current_playing_genre)
+        detailsTextView = view.findViewById(R.id.current_file_details)
```

Comment:
I prefer:
```
bitrateFormatTextView = view.findViewById(R.id.current_playing_bitrate_format)
```

It would be necessary to change the name of the variables everywhere obviously. :wink:

Reply:
Refactoring makes the job, so thats easy. ;-) Done.